buildpipeline/DotNet-Trusted-Publish-Symbols.json

```diff
@@ -46,6 +46,7 @@
       "alwaysRun": false,
       "displayName": "Extract symbol packages; if release branch, archive",
```

Comment:
How is this related to myget? I don't see us pushing anything to myget in this step unless it is happening under the covers.

Reply:
it's related in that if we don't publish to myget but we publish the symbols to MS symbol servers we'll have posted symbols that end-users can't get to.  Mostly, the idea is to not waste time and resources with symbols noone is likely to need, since the flat blob storage will have symbol packages included.

Should I remove it?

Reply:
I don't know that I would condition this on the myget publish condition, as I think we would want to do this if we do any sort of publish. If we find it interesting we could have a NoPublicPublish option or something that could cover this if we need.